services/src/main/java/org/keycloak/services/resources/WelcomeResource.java

```diff
@@ -302,4 +310,16 @@ private void csrfCheck(final MultivaluedMap<String, String> formData) {
         }
     }
 
+    protected String getThemeProperty(String propertyName, String defaultValue) {
```

Comment:
This is a minor behavioral change to the existing logic - catching rather than throwing an IOException. Is this method for subclasses? And / or does it be long as a utility method on Theme?